src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1138,14 +1135,27 @@ private TokenValidationResult ValidateTokenPayload(JsonWebToken jsonWebToken, To
         /// <summary>
         /// Validates the JWT signature.
         /// </summary>
-        private static JsonWebToken ValidateSignature(string token, TokenValidationParameters validationParameters)
+        private static JsonWebToken ValidateSignature(string token, TokenValidationParameters validationParameters, BaseConfiguration configuration)
         {
             if (string.IsNullOrWhiteSpace(token))
                 throw LogHelper.LogArgumentNullException(nameof(token));
 
             if (validationParameters == null)
                 throw LogHelper.LogArgumentNullException(nameof(validationParameters));
 
+            if (validationParameters.SignatureValidatorWithConfiguration != null)
+            {
+                var validatedToken = validationParameters.SignatureValidatorWithConfiguration(token, validationParameters, configuration);
```

Comment:
In the other PR, we suggested a name change to: SignatureValidatorUsingConfiguration instead of SignatureValidatorWithConfiguration. With -> Using.

Reply:
All instances of WithConfiguration have been replaced with UsingConfiguration